### PR TITLE
Remove `isConnected` polyfill

### PR DESCRIPTION
**Pull request by ChrisAntaki**
https://caniuse.com/mdn-api_node_isconnected

### CODE REVIEW

File: src/components/friendly-iframe.js [hunk 1]
```diff
@@ -94,6 +94,6 @@ export class FriendlyIframe {
    * @return {boolean}
    */
   isConnected() {
```
Comment (fikus) [inline src/components/friendly-iframe.js:96]: Add unit tests for this method?

File: src/utils/dom.js [hunk 2]
```diff
@@ -17,18 +17,9 @@
...
-  'rel': 'stylesheet',
-  'type': 'text/css',
-};
-
 /** @const {string} */
 export const styleType = 'text/css';
 
-/** @const {string} */
-export const styleExistsQuerySelector = 'link[rel=stylesheet][href]';
-
```
Comment (fikus) [inline src/utils/dom.js:31]: These changes seem unrelated.

Comment (ChrisAntaki) [reply src/utils/dom.js:31]: Oops, I'll create a PR for these changes

Comment (ChrisAntaki) [reply src/utils/dom.js]: Here's a PR that remove these extra DOM constants
https://github.com/subscriptions-project/swg-js/pull/2504

File: src/components/friendly-iframe.js (hunk 1 above)
Comment (fikus) [reply src/components/friendly-iframe.js:96]: Hmm.. There's an opportunity to split that into more specific tests that are unit testing isolated behavior. As is, I don't think it's testing the behavior of isConnected specifically, ie that the method returns the isConnected value of its element.

Comment (ChrisAntaki) [reply src/components/friendly-iframe.js:96]: Here's a new test:
https://github.com/subscriptions-project/swg-js/pull/2501/commits/a030094db15743c43ca5dd2abe964c58617f563e

**Review by fikus — APPROVED** [review]
